{"name": "Тест принятия заказа с неверным id курьера/заказа", "status": "passed", "description": "Если передать неверный id курьера/заказа, запрос вернёт ошибку Not Found", "steps": [{"name": "Отправляем запрос на принятие заказа", "status": "passed", "parameters": [{"name": "order_id", "value": "277504"}, {"name": "courier_id", "value": "4338817"}], "start": 1715023388401, "stop": 1715023389048}, {"name": "Отправляем запрос на авторизацию курьера", "status": "passed", "parameters": [{"name": "payload", "value": "{'login': 'yfenwdkldx', 'password': 'gornsxnzvq'}"}], "start": 1715023389048, "stop": 1715023389729}, {"name": "Отправляем запрос на удаление курьера", "status": "passed", "parameters": [{"name": "courier_id", "value": "301403"}], "start": 1715023389729, "stop": 1715023390389}], "parameters": [{"name": "parameter_number", "value": "1"}], "start": 1715023388401, "stop": 1715023390396, "uuid": "d091f68f-adab-4bc8-aac7-21d9b11a95b8", "historyId": "95d0217eea775236bfb2e0fe7c2919d8", "testCaseId": "469674aa16559dae50bc69cfbf16945a", "fullName": "tests.test_orders_accept.TestOrdersAccept#test_accept_order_wrong_id_return_not_found_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_orders_accept"}, {"name": "subSuite", "value": "TestOrdersAccept"}, {"name": "host", "value": "DESKTOP-9PI1IG9"}, {"name": "thread", "value": "9108-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_orders_accept"}]}